common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -39,6 +39,41 @@ killall_program() {
     done
 }
 
+is_wireless_iface() {
+        iwconfig "$1" > /dev/null 2>&1
+}
+
+is_bridge_iface() {
+    [ -d "/sys/class/net/$1/bridge" ] && return 0
+    return 1
+}
+
+platform_init() {
```

Comment:
I really prefer to have a separate commit that just moves the code around. That makes it a lot easier to see what really has changed.

Reply:
I keep forgetting. Next time, promise:)